src/IssueViz.UnitTests/Editor/LocationTagging/IssueLocationStoreAggregatorTests.cs

```diff
@@ -36,13 +36,24 @@ namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Editor.LocationTag
     public class IssueLocationStoreAggregatorTests
     {
         [TestMethod]
-        public void MefCtor_CheckExports()
+        public void MefCtor_CheckIsExported()
```

Comment:
New test - this type effectively has no required types since it is `ImportMany`.